backend/graminsta/post/models.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: UTF-8 -*-
+"""This models.py defines the model of our post app"""
+
+from django.db import models
+from django.contrib.auth import models as auth_models
+
+
+class Post(models.Model):
+    """
+    A Post is usually auth_models.User generated which contains information
+        like an image,and some description to the image
+    """
+    publisher = models.ForeignKey(
+        auth_models.User,
+        on_delete=models.CASCADE,
+        related_name="published_posts")
+    time = models.DateTimeField(auto_now=True)
+    marked_user = models.ManyToManyField(
+        auth_models.User, related_name="marked_post")
+    image_url = models.TextField()
```

Comment:
I thinnk it should be ImageField instead of TextField. See here: https://docs.djangoproject.com/en/3.0/ref/models/fields/#imagefield

Reply:
+1